### ISSUE TITLE
Mute automatic speech feedback

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#481]
**Reported by tvalusek on 2009-11-26 08:38**
Some screenreaders for mobile devices have an extremely useful feature for partially-sighting users - an option to mute automatic feedback issued during normal device use. However, all commands involving a dedicated "screenreader" key operate as usual, including voice feedback. That means one can silently navigate menus, and when there is an item with incomplete title shown, a screenreader command for reading current item can be used without re-enabling speech.

I'd like to see similar behavior in NVDA - a mode with no spoken feedback on ordinary key presses (including command keys such as TAB, Enter, Space etc.), but all NVDA commands involving NVDA key operating as usual. This could be useful for Braille users too. Virtual buffers should be automatically engaged in form mode when auto-feedback is muted, so that they don't interfere with common control of e.g. web browsers, PDF readers etc.

**Comment by leonardder** [comment]
This sounds a bit like sleep mode, but slightly different.
@JosephSL: Since you commented, what do you think about this issue?

**Comment by Adriani90** [comment]
I think sort of a solution has been implemented in Fusion from Freedom Scientific. This works in connection with magnification feature.

**Comment by Adriani90** [comment]
I suggest assigning some gestures to fire speech announcement on current NVDA cursor position, current mouse position and current system focus position. Let's say I disable speech by NVDA+s, and then I navigate with the keyboard through a menu, I could press NVDA+ctrl+tab and NVDA would announce my focus. NVDA+left mouse click would announce the object under the mouse cursor. NVDA+shift+ctrl+tab would announce current object at NVDA cursor.

**Comment by josephsl** [comment]
Hi, instead of defining completely new gestures, I suggest using a flag in scripts to signal if this is subject to announcement or not. Thanks.

**Comment by bdorer** [comment]
This would be a great feature for visually impaired persons using magnification.

**Comment by Adriani90** [comment]
At least the part with the browse mode not being loaded automatically is already solved, you can turn off automatic load browse mode on page load in browse mode settings. However, I think there is no gesture implemented for that specific setting yet.
